advisories/0011.md

```diff
@@ -0,0 +1,57 @@
+### Mirantis Kubernetes Engine (MKE) and CVE-2024-21626: Understanding the Impact and Mitigation
+#### Release Date
+2024-02-08
+#### Overview
+This advisory addresses the impact of CVE-2024-21626, a critical vulnerability identified within runc, and its implications for the Mirantis Kubernetes Engine (MKE). Given that MKE relies heavily on the Mirantis Container Runtime (MCR), this vulnerability directly impacts its operational integrity. We aim to clarify the situation, explain the mitigation steps, and provide actionable guidance for MKE users.
+
+### Vulnerability Information
+#### CVE Identifier
+CVE-2024-21626
+#### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+#### CWEs
+CWE-403, CWE-668
+
+#### Affected Product
+* MKE deployment requires installation of MCR.  Older versions of MCR (<= 23.0.9) contain a vulnerability in the underlying runc component. Please see [the MCR advisory](https://github.com/Mirantis/security/blob/main/advisories/0010.md) for details. Any deployment of MKE with MCR version <= 23.0.9 must upgrade to MCR 23.0.9-1 or later to mitigate this risk.
+
+* No MKE Image Update Needed for CVE-2024-21626
+  
+   While CVE-2024-21626 might appear in vulnerability scans of your MKE images due to a dependent library, an update to MKE images is not currently required. MKE's architecture mitigates the vulnerability, and there's no known exploit path within MKE. We understand this discrepancy can be confusing. To address this and enhance transparency, we're actively working to remove unnecessary references to the vulnerability from future MKE image scans. This update is planned for upcoming MKE releases.
```

Comment:
> MKE's architecture mitigates the vulnerability, and there's no known exploit path within MKE.

That statement could be easily misinterpreted as "MKE protects you from the runc vulnerability." MKE _components_ are not vulnerable to this particular exploit, but MKE _does not protect users from harm_ if they ran a malicious container image with a vulnerable runc installed.

Reply:
Let me rewrite this paragraph